falcon/util/deprecation.py

```diff
@@ -22,12 +22,17 @@
 
 
 __all__ = (
+    'RemovedError',
     'DeprecatedWarning',
     'deprecated',
     'deprecated_args',
 )
 
 
+class RemovedError(RuntimeError):
```

Comment:
Would it make sense to inherit from `AttributeError` instead? (If it is possible to subclass it.)

Reply:
I think it would make sense

Reply:
ah, good point. let me do that.